docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,141 @@
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Set up a test
+
+Since project tests share the same canvas as models, you can set them up the same way you would develop and run a model.
+
+### Develop a test
+
+To develop a project test, start by opening a project:
+
+1. Under the Project Browser, click **Add Test**.
+
+   ![Add a new test](img/project-test-add-new.png)
+
+2. Enter your **Test Name**. Project tests are saved to a tests SQL file in the Git directory by default.
+
+3. Click **OK** to create your new test.
+
+4. Create your test model by dragging and dropping Gems to your visual canvas. Connect them to your Data Test. You can also write your test on the Code view. You can use as many models and other database objects as you’d like in your test.
+
+   ![Project test canvas](img/project-test-canvas.png)
+
+### Run a test
+
+After you’ve developed your project test and executed all of your models, you can run the test.
+
+1. Click the **Play** button on either the canvas or an individual Gem to execute your test. The table input to the data test Gem is what’s tested.
+
+   1. Run a whole test. Clicking the canvas **Play** button executes the complete test and shows the interim data after the Data Test Gem. Additionally, it displays the final test status, succeeded, warning, or failed, in a summary.
+
+   2. Run a partial test. Clicking the Gem **Play** button executes the test SQL up to a particular Gem, and displays that Gem’s data on the output. This option doesn’t execute the data test.
+
+2. Click **See Run Details** to view the test Summary. Depending on the outcome of the test, the icon displays a different color.
+
+   ![See Run Details](img/project-test-run-details.png)
+
+   You can click to expand the test logs in the Summary to view the dbt logs.
+
+   ![View test summary](img/project-test-summary.png)
+
+In addition to the previous relational integrity test, you can create tests to check that the total payment amount from customers is positive or simply check that all of your tables are still accessible. You can test any series of transformation Gems, because a project test simply asserts that the resulting table meets a certain criteria.
+
+## Fix a failed test
+
+If your project test fails, check your test model for any changes.
+
+![Failed test](img/project-test-fail.png)
+
+- Click the test output to see your failed rows.
+
+Some examples of test failure causes include:
+
+- Happy path: There is now a row in the table → you have detected the thing you were trying to detect!
+- The table no longer exists since one or more input data sources were deleted or inaccessible
+- Dataset schema changes, such as in the column or datatype
+- The seed in the test is different from what’s in your warehouse
+- Code releases that caused a breaking change
```

Comment:
I would remove the following lines unless Engineering specifically says they should be included here. 

"- Dataset schema changes, such as in the column or datatype
- The seed in the test is different from what’s in your warehouse
- Code releases that caused a breaking change"

Reply:
Done.

Added a couple of new failure causes in their place.